packages/venia-concept/src/actions/cart/asyncActions.js

```diff
@@ -350,6 +350,9 @@ export const getCartDetails = (payload = {}) => {
             // for each item in the cart, look up its image in the cache
             // and merge it into the item object
             // then assign its options from the totals subResource
+            // TODO: If we don't have the image in cache we should probably try
+            // to find it some other way otherwise we have no image to display
+            // in the cart and will have to fall back to a placeholder.
```

Comment:
If a user signs in on a different device after adding an item we have no image to display in the cache. I fixed it by adding some conditional usage in this PR but I also added this TODO because we should probably find a way to re-request a non-cached image for an item added from another device or before a cache clear.